public/styles/style.css

```diff
@@ -1,3 +1,106 @@
+html {
```

Comment:
Yes, but you have committed it to the code base. The idea is that you only commit source code..

eg if using gh-pages, you would only commit this file to gh-pages branch, and would not add it to master.. Does that make sense?